frontend/packages/policy-editor/src/components/ExpandablePolicyCard/ExpandablePolicyCard.tsx

```diff
@@ -199,8 +199,14 @@ export const ExpandablePolicyCard = ({
   /**
    * Displays the actions
    */
-  const displayActions = policyRule.actions.map((a, i) => {
-    return <ActionAndSubjectListItem key={i} title={a} onRemove={() => handleRemoveAction(i, a)} />;
+  const displayActions = policyRule.actions.map((actionId, i) => {
+    return (
+      <ActionAndSubjectListItem
+        key={i}
+        title={t(`policy_editor.action_${actionId}`)}
+        onRemove={() => handleRemoveAction(i, actionId)}
```

Comment:
Out of scope for this pull request, however it could be feasible to simplify the code by eliminating the index and relying on the `actionId` only. 😊 
```suggestion
        onRemove={() => handleRemoveAction(actionId)}
```